### DISCUSSION TITLE
How can I broadcast a signed transaction with a simple http request?

### DISCUSSION THREAD

**Discussion by kshitijc1506** [aptos-labs/aptos-developer-discussions#101] [Questions] [answered]
Asked by `glant` on Discord (#dev-discussion)

> I have sign a transaction use TS sdk, I can get the hex raw data,  how can I send it to the network use a simple http request ?
> const senderAuthenticator = aptos.transaction.sign({
>         signer: ed25519Account,
>         transaction: simpleTransaction,
>     });
>     console.log(senderAuthenticator.bcsToHex().toString());
> 
> how can I broadcast the signed tx whit a simple http request?

**Comment by kshitijc1506** [comment, marked as answer]
To submit the signed transaction on-chain, you can use the TypeScript SDK. 

Have a look at this [section](https://aptos.dev/sdks/ts-sdk/transaction-builder/#simple-transaction) of the docs.  

If you want to use a http request, you can refer to the [API specification](https://aptos.dev/nodes/aptos-api-spec/#/). Navigate to Transactions and have a look at the POST request to submit a transaction.

**Reply by glant509** [reply]
thank you! It has been resolved
